PENDING.md

```diff
@@ -26,9 +26,8 @@ BREAKING CHANGES
     * [x/stake] [#1676] Revoked and jailed validators put into the unbonding state
     * [x/stake] [#1877] Redelegations/unbonding-delegation from unbonding validator have reduced time
     * [x/stake] \#2040 Validator operator type has now changed to `sdk.ValAddress`
-      * A new bech32 prefix has been introduced for Tendermint signing keys and
-        addresses, `cosmosconspub` and `cosmoscons` respectively.
-    
+          * A new bech32 prefix has been introduced for Tendermint signing keys and addresses, `cosmosconspub` and `cosmoscons` respectively.
```

Comment:
unnecessary/ undesirable

Reply:
https://github.com/cosmos/cosmos-sdk/pull/2103/files if we do introduce new bech32's, it would be `cosmosval` and `cosmosvalpub`, not `cosmoscons*` just fyi.

Reply:
I think maybe there's something wrong with the sdk impl right now... i don't think the validator operator address should be anything different than a normal address.

Reply:
@jaekwon @rigelrozanski  Currently `BechValidator` is not using `bech32` prefix for `sdk.ValAddress,  just hex. Also its `PubKey` value uses `cosmosconspub` prefix. Is that ok ? Because I thought it was supposed to be using `cosmosvalpub` instead